tool/teleport/common/teleport.go

```diff
@@ -366,6 +369,14 @@ func onConfigDump(flags dumpFlags) error {
 		flags.LicensePath = filepath.Join(defaults.DataDir, "license.pem")
 	}
 
+	if flags.KeyFile != "" && !filepath.IsAbs(flags.KeyFile) {
```

Comment:
Can we figure out the path ourselves using something like `filepath.Abs` instead of asking the user? Same for cert file.

Reply:
will give it a try